Description: Use FHS compliant path for sntp kod database
 /var/db/ntp-kod is hardcoded in the sources and not permitted by FHS
 .
 The path is also present in sntp/sntp-opts.c and sntp/sntp.1sntpman, which
 get regenerated by autogen at build time
Author: Bernhard Schmidt <berni@debian.org>
Bug: http://bugs.ntp.org/show_bug.cgi?id=3396

--- a/sntp/sntp-opts.def
+++ b/sntp/sntp-opts.def
@@ -113,7 +113,7 @@
   value		= K;
   arg-type	= file;
   arg-name	= "file-name";
-  arg-default	= "/var/db/ntp-kod";
+  arg-default	= "/var/lib/sntp/kod";
   descrip	= "KoD history filename";
   doc		= <<- _EndOfDoc_
 	Specifies the filename to be used for the persistent history of KoD
--- a/sntp/sntp.man.in
+++ b/sntp/sntp.man.in
@@ -154,7 +154,7 @@
 \f\*[I-Font]file\-name\f[]
 for this option is:
 .ti +4
- /var/db/ntp-kod
+ /var/lib/sntp/kod
 .sp
 Specifies the filename to be used for the persistent history of KoD
 responses received from servers.  If the file does not exist, a
